src/towncrier/test/test_build.py

```diff
@@ -1248,6 +1248,49 @@ def test_default_start_string_markdown(self, runner):
 
         self.assertEqual(expected_output, output)
 
+    @with_project(
+        config="""
+        [tool.towncrier]
+        directory = "changes"
+        filename = "NEWS.md"
+        version = "1.2.3"
+        """
+    )
+    def test_markdown_no_name_title(self, runner):
+        """
+        Markdown renders a title even when there's no project name
```

Comment:
I think that this is a better description for the test.

```suggestion
        When configured with an empty `name` option,
        the default template used for Markdown,
        renders the title of the release note with just
        the version number and release date.
```